pkg/features/kube_features.go

```diff
@@ -1017,7 +1018,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	RotateKubeletServerCertificate: {Default: true, PreRelease: featuregate.Beta},
 
-	SeccompDefault: {Default: true, PreRelease: featuregate.Beta},
+	SeccompDefault: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.29
```

Comment:
sweep and remove checks of `...Enabled(features.SeccompDefault)` now that this is locked true

Reply:
Ah yeah makes sense, I removed the checks.